compiler_gym/envs/llvm/make_specs.py

```diff
@@ -11,10 +11,17 @@
 # TODO: As we add support for more compilers we could generalize this script
 # to work with other compiler services rather than hardcoding to LLVM.
 import sys
+import types
 from pathlib import Path
 
-from compiler_gym.envs.llvm.llvm_env import LlvmEnv
-from compiler_gym.util.runfiles_path import runfiles_path
+# Avoids circular dependency during specs.py generation,
+# because it is imported from compiler_gym.envs.llvm before being generated.
```

Comment:
Same here:

```suggestion
# TODO(github.com/facebookresearch/CompilerGym/issues/506): Avoids circular 
# dependency during specs.py generation, because it is imported from 
# compiler_gym.envs.llvm before being generated.
```

Reply:
Done.